csharp/OnnxRuntime.CSharp.proj

```diff
@@ -61,12 +70,48 @@ CMake creates a target to this project
     </Exec>
   </Target>
 
+   <Target Name="ObtainPackageVersion" BeforeTargets="CreatePackage">
```

Comment:
This target is copied from the 'Microsoft.ML.OnnxRuntime.csproj' since we need to decide upon the version number here as the version number is common across the native and the managed nugets